ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1630,11 +1650,33 @@ def test_paddle_rad2deg(
 )
 def test_paddle_fmax(
```

Comment:
Hello @elabongaatuo , first nice work 😄  put can you write the test again or revert the last commit as you commit it with out resolve conflicts that leads to divide the test code and it will not work so please look at it again

Reply:
Thank you @paulaehab , let me resolve that

Reply:
hi @elabongaatuo  after you finished it request review again from me